pkg/crossplane/metadata.go

```diff
@@ -46,6 +47,26 @@ const (
 	SLALabel = SynToolsBase + "/sla"
 )
 
+const (
+	// SLAPremium represents the string for the premium SLA
+	SLAPremium = "premium"
+	// SLAStandard is an empty string
+	SLAStandard = ""
```

Comment:
I'm a fan to make things explicit. Would it make sense to write "standard" into this field?

Reply:
@tobru I'd prefer that as well but ATM the crossplane plans without premium SLA have the field empty - I merely reflect that fact in the code here. Not sure why it's empty and not `standard` - maybe @srueg can shed some light on this?

Reply:
Yes, absolutely. I'll add `standard` to the plans :+1:

Reply:
Done https://git.vshn.net/swisscompks/component-spks-crossplane/-/merge_requests/6

Reply:
thanks :+1: 

fixed.